packages/fxa-payments-server/src/jest.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  preset: 'ts-jest',
+  moduleNameMapper: {
+    '\\.(css|scss)$': '<rootDir>/../node_modules/jest-css-modules',
```

Comment:
The alternative to this is to create a mock file containing an empty object for CSS/SCSS files to hit.

It does set us up for this, though: https://github.com/mozilla/fxa/issues/2139